tasks/main.yml

```diff
@@ -81,7 +81,7 @@
     name: "Request new certificates when needed and remove unused ones"
     job: "/usr/local/sbin/manage_certs.py @{{ LOAD_BALANCER_MANAGE_CERTS_CONF }}"
     cron_file: manage-certs
-    minute: "*/5"
+    minute: "*/{{ LOAD_BALANCER_MANAGE_CERTS_INTERVAL }}"
```

Comment:
Cron mails are sent to root by default, and when this role is included in our full setup, root email will be [forwarded to `OPS_EMAIL`](https://github.com/open-craft/ansible-forward-server-mail/blob/master/templates/aliases#L1) and [`LOAD_BALANCER_OPS_EMAIL` is also set to `OPS_EMAIL`](https://github.com/open-craft/deployment-deploy-services/blob/master/group_vars/load-balancer/public.yml#L11).  In the integration tests, the load-balancer role is used in isolation, so mail isn't forwarded.